templates/README.md

```diff
@@ -12,6 +12,13 @@ The documents declared in this repository are:
 
 - ...
 
+## Deployment
+
+For deployment to occur correctly, you need to setup [Github secrets](https://docs.github.com/en/enterprise-cloud@latest/actions/security-guides/encrypted-secrets) with the following names
+
+- `SERVER_FINGERPRINT` Generate it on the target server with `ssh-keygen -t rsa -b 4096 -C "$email" and then cat $nameofthefile.pub >> authorized_keys`
+- `SERVER_SSH_KEY` Generate with `ssh-keyscan -H $targetServerIpAddress`
```

Comment:
Looks like the two names got their definitions reversed 😉

Let's make the key setup as simple as possible. I'm not sure I understand why we would need the email as a comment to the key 🤔 

```suggestion
- `SERVER_FINGERPRINT`: obtained with `ssh-keyscan -H $serverAddress`.
- `SERVER_SSH_KEY`: a private SSH key allowed to connect to your server.

> You can for example generate one on your server with `ssh-keygen -q -N "" -f ~/.ssh/ota-deploy && cat ~/.ssh/ota-deploy.pub >> authorized_keys`, and store the contents of `~/.ssh/ota-deploy` as the `SERVER_SSH_KEY` secret.
```

Reply:
Sure, I usually create keys like this but we can remove it

Reply:
It's just that if we mention a `$email` variable, we have to explain what it is and to what it should be set.